### PR TITLE
fix: change selenium to playwright

### PR DESCRIPTION
**Pull request by VitalyTolstikA1QA**

### CODE REVIEW

File: test_project/tests/ui/test_spotify_tracks.py [hunk 1]
```diff
@@ -11,16 +13,14 @@
...
     ]
 )
-def test_spotify_tracks(browser, artist, song_name):
-    home_page = HomePage(browser)
-    home_page.open('https://spotify.com')
-    assert home_page.is_cookies_button_displayed(), 'Accept cookies button is not displayed'
-    home_page.click_cookies_button()
-    home_page.wait_until_cookies_button_is_not_displayed()
-    assert not home_page.is_cookies_button_displayed(), 'Accept cookies button is displayed'
+def test_spotify_tracks(page: Page, artist, song_name):
```
Comment (mialeska) [inline test_project/tests/ui/test_spotify_tracks.py]: could we make Page a singleton stored in some container and initialized through fixture? Could we avoid passing it to every constructor?

File: test_project/pages/home_page.py [hunk 2]
```diff
@@ -1,34 +1,22 @@
...
-        super().__init__(browser)
-
-    def __get_cookies_button(self) -> WebElement:
-        return self.find_element(cookies_button_selector)
-
-    def click_cookies_button(self):
-        self.__get_cookies_button().click()
+    def __init__(self, page: Page):
+        super().__init__(page)
+        self.cookies_button = page.get_by_role("button", name="Accept Cookies")
```
Comment (mialeska) [inline test_project/pages/home_page.py]: items would be searched during the page init, isn't it? would the init fail if any of them missing?

File: test_project/tests/ui/test_spotify_tracks.py (hunk 1 above)
Comment (VitalyTolstikA1QA) [reply test_project/tests/ui/test_spotify_tracks.py]: fixed

File: test_project/pages/home_page.py (hunk 2 above)
Comment (VitalyTolstikA1QA) [reply test_project/pages/home_page.py]: fixed

File: test_project/tests/ui/conftest.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from framework.data_processors.patterns.page_singleton import PageSingleton
+from playwright.sync_api import Page
+
+
+@pytest.fixture
+def init_page(page: Page):
+    yield PageSingleton.get_instance(page)
```
Comment (mialeska) [inline test_project/tests/ui/conftest.py:9]: Cool! You can think about the parallelization in some next task

**Review by mialeska — APPROVED** [review]

**Pull request merged**